_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy.cpp

```diff
@@ -539,6 +540,20 @@ void Legacy::SetInherited(ParamInheritance& par)
         INHERIT_OPT(VideoFullRange);
         INHERIT_OPT(ColourDescriptionPresent);
     });
+
+    par.m_ebInheritDefault[MFX_EXTBUFF_AVC_TEMPORAL_LAYERS].emplace_back(
+        [](const mfxVideoParam& /*parInit*/
+            , const mfxExtBuffer* pSrc
+            , const mfxVideoParam& /*parReset*/
+            , mfxExtBuffer* pDst)
+    {
+        INIT_EB(mfxExtAvcTemporalLayers);
+        for (mfxU32 i = 0; i < 7; i++)
```

Comment:
Could 7 be changed to some define or at least add comment why 7.

Reply:
Thanks @lakulako , comments are added.

Reply:
Fine, then may be add also something like "HW API limitation"?

Reply:
Sure. The value comes from HEVC spec, I have explained it in the comments.